packages/apputils-extension/src/index.ts

```diff
@@ -564,8 +568,13 @@ const sessionDialogs: JupyterFrontEndPlugin<ISessionContextDialogs> = {
 const utilityCommands: JupyterFrontEndPlugin<void> = {
   id: '@jupyterlab/apputils-extension:utilityCommands',
   requires: [ITranslator],
+  optional: [IMainMenu, ICommandPalette],
```

Comment:
You don't use the menu so `IMainMenu` is unneeded - this is what creates the error as the `activate` function gets as arguments `[app, ...requires, ...optional]`. So in the current code, the variable `palette` received `IMainMenu` and not `ICommandPalette`

```suggestion
  optional: [ICommandPalette],
```